examples/model_compress/models/cifar10/resnet.py

```diff
@@ -0,0 +1,115 @@
+import torch
+import torch.nn as nn
```

Comment:
what is the difference between this file and torchvision.models.resnet ?

Reply:
> what is the difference between this file and torchvision.models.resnet ?
The only difference is here: https://github.com/suiguoxin/nni/blob/benchmark/examples/model_compress/models/cifar10/resnet.py#L69-L70 
We can not use torchvision.models.resnet because these args are not configurable.